index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+    <h1 id="title">Malaria Research</h1>
+<p id="description">Thank you for taking time to fill in this survey!</p>
+<form id="survey-form">
+
+  <p>Personal Details</p>
+  <label for="name" id="name-label">Name:</label>
+  <input type="text" required placeholder="name" id="name">
+  <br/>
+  <label for="email" id="email-label">Email:</label>
+  <input type="email" required placeholder="email" id="email"/><br/>
+  <label for="number" id="number-label">Number:</label>
+  <input type="number" placeholder="number" id="number" min="0" max="100"/><br/>
+
+  <p>Where is your location?</p>
```

Comment:
- [ ] Great job asking politely for a change! 💯 However, remember that a good review should be clear and include an explanation. Please add those! Saying that "The question needs more attention" is very ambiguous. 🤓 Please, consider including a better explanation as well as correcting grammar and spelling errors! 🛠️ 

For example:
>* This is a well-explained request: *Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible*
>* This request lacks a "why": *Why don't you use an article tag here instead of a div?*

>* This is a clear comment: *The README's "Built With" and "Getting Started" sections are currently empty. Why don't you add content, listing the technology used in the project and how to run it, to them so your README is more informative and professional?*
>* This is a vague comment: *The README is not descriptive, why don't you make it better?*